src/main/java/io/boomerang/service/refactor/TaskServiceImpl.java

```diff
@@ -549,6 +563,17 @@ private void finishWorkflow(ActivityEntity activity) {
     final long duration = finishDate.getTime() - activity.getCreationDate().getTime();
     activity.setDuration(duration);
 
+    List<TaskExecutionEntity> taskExecutions =
+        taskActivityService.findTaskActiivtyForActivity(activity.getId());
+    if (activity.getOutputProperties() == null) {
+      activity.setOutputProperties(new LinkedList<>());
+    }
+
+    for (TaskExecutionEntity taskExecution : taskExecutions) {
+      if (taskExecution.getOutputProperties() != null) {
+        activity.getOutputProperties().addAll(taskExecution.getOutputProperties());
```

Comment:
Hey @amhudson would there be a possibility of incorrectly overriding the same parameter by chance? if the same key is used across multiple Set Result Parameter tasks...

Reply:
Ok. So is that another problem. What happens when you try to reference or use them? 🤔

Reply:
I don't think we have any validations to prevent the same key(s) from being used

Reply:
Alright, so what I am hearing is that its not going backwards from what we have in use of the workflow results. Its just not working how it should in the long term. Thats ok, we can create a backlog item for it.

Reply:
https://github.com/boomerang-io/flow.service.workflow/issues/241